test/SavingsDaiTokenWrapper.t.sol

```diff
@@ -41,4 +48,40 @@ contract SavingsDaiTokenWrapperTest is BaseTokenWrapperTest {
       'Unexpected TOKEN_IN allowance'
     );
   }
+
+  function testBorrow() public {
```

Comment:
this can be moved to the base contract too

Reply:
updated in 340414fd777c2012469137465a4a20aab48fd5ef